### DISCUSSION TITLE
Issue adding another Sv06 Plus

### DISCUSSION THREAD

**Discussion by Rpm49** [bassamanator/Sovol-SV06-firmware#98] [General]
Inside my files I am replacing the files you mentioned that are pointing to my new printer. Home/pi/whatever printer. My path is home/rpm/Sv06p1_data and home/rpm/Sv06p2_data. I can copy/paste the correct files/change the files to get sv06p1 working, but when i set up the sv06p2 printer I get mcu errors. I have tried path and id while verifying it's the correct connection. Before I research more, is there any reason why the 2nd printer isnt working (files wise). I have your sv06 running at the same time as well. Ubuntu. so 06 and 06P is working fine, i just cant add a second due to errors. 

Thanks for your wisdom

**Comment by bassamanator** [comment]
My friend you have to use something like [KIAUH](https://github.com/dw-0/kiauh) to host multiple printers on the same host.

**Reply by Rpm49** [reply]
Sorry for the long responce, the mobo i recieved apparently has MCU connection/flashing issues. I swapped it back with the old board and setup is all fine. Thanks and sorry for jumping to asking, was just really strange. Got a 06 and x2 sv06+'s going now. Thank you so much for your help <3

**Reply by bassamanator** [reply]
Pleasure.
